Formula/ntfy.rb

```diff
@@ -0,0 +1,42 @@
+class Ntfy < Formula
+  desc "Send push notifications to your phone or desktop via PUT/POST"
+  homepage "https://ntfy.sh/"
+  url "https://github.com/binwiederhier/ntfy.git", tag: "v2.3.1", revision: "a75fb08ef138d67599499c1f78628a0b35fcef54"
+  license any_of: ["Apache-2.0", "GPL-2.0-only"]
+  head "https://github.com/binwiederhier/ntfy.git", branch: "main"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
```

Comment:
```suggestion
```
I don't think you need this. To see, check the output of `brew livecheck ntfy` with and without this block.

Reply:
Did you forget the `livecheck` block? I checked myself and it doesn't look like it's needed.

Reply:
I tested now locally without the block, it is able to livecheck even without it